subcommands/keys/tuf_updates.go

```diff
@@ -0,0 +1,42 @@
+package keys
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var tufUpdatesCmd = &cobra.Command{
+	Use:   "updates",
+	Short: "Manage updates to the TUF root for your factory (expert mode)",
+	Long: `These sub-commands allow you to transactionally stage and apply changes
+to your Factory's TUF private keys in a granular way familiar for TUF experts.
+
+The TUF updates transaction starts by running the "fioctl keys tuf updates init" command.
+That command returns a unique secure Transaction ID which is then required for other actions.
+The admin initiating the transaction should save that TXID for the timespan of the transaction.
```

Comment:
What happens if `txid` is lost? Is the update expires somehow or it can be cancelled without a need in `txid`?

Reply:
I see that `cancel` can be performed without `txid`. iirc, only `admin` can perform `cancel`?

Reply:
Yes